scripts/export-reportbacks-to-rogue.php

```diff
@@ -38,20 +36,24 @@
 
   // Only try to send to Rogue if we have a Northstar ID
   if (isset($northstar_id)) {
+    // Grab the image
+    $file = dosomething_helpers_get_data_uri_from_fid($rb->fid);
+
     $data = [
       'northstar_id' => $northstar_id->id,
       'drupal_id' => $rb->uid,
       'campaign_id' => $rb->nid,
       'campaign_run_id' => $rb->run_nid,
       'quantity' => $rb->quantity,
       'why_participated' => $rb->why_participated,
-      'file' => dosomething_helpers_get_data_uri_from_fid($rb->fid),
+      'file' => $file,
       'caption' => $rb->caption,
       'source' => $rb->source,
       'remote_addr' => $rb->remote_addr,
       'status' => dosomething_rogue_transform_status($rb->status),
       // Tell rogue not to try to forward the reportback back to phoenix.
       'do_not_forward' => TRUE,
+      'default_image' => isset($file) ? NULL : TRUE,
```

Comment:
oh oh oh yes I think because if it is an update with no image I'm not sure Rogue could tell the difference between that and a missing image. I guess if the file helper returns `false` instead of `null` then Rogue could see if `file` isset and is false?